managed/models/database.go

```diff
@@ -1057,30 +1060,66 @@ func setupFixture1(q *reform.Querier, username, password string) error {
 		return err
 	}
 
+	address := strings.Split(params.Address, ":") // TODO: find better solution
+	if params.Address != DefataultPostgreSQLAddr {
+		if node, err = CreateNode(q, RemoteNodeType, &CreateNodeParams{
+			NodeName: "pmm-server-db",
+			Address:  address[0],
+		}); err != nil {
+			return err
+		}
+	}
+
+	port := uint64(5432)
+	if len(address) == 2 {
+		port, err = strconv.ParseUint(address[1], 10, 16)
+		if err != nil {
+			return err
+		}
+	}
+
 	// create PostgreSQL Service and associated Agents
 	service, err := AddNewService(q, PostgreSQLServiceType, &AddDBMSServiceParams{
 		ServiceName: PMMServerPostgreSQLServiceName,
 		NodeID:      node.NodeID,
-		Address:     pointer.ToString("127.0.0.1"),
-		Port:        pointer.ToUint16(5432),
+		Database:    params.Name,
+		Address:     &node.Address,
+		Port:        pointer.ToUint16(uint16(port)),
 	})
 	if err != nil {
 		return err
 	}
+
+	tls := params.SSLMode != "disable"
+	tlsSkipVerify := params.SSLMode == "disable" || params.SSLMode == "verify-ca"
+	var psqlOps *PostgreSQLOptions
+	if tls {
+		psqlOps = &PostgreSQLOptions{
+			SSLCa:   params.SSLCAPath,
+			SSLCert: params.SSLCertPath,
+			SSLKey:  params.SSLKeyPath,
```

Comment:
we should read these files and store content in DB.

Reply:
redone to store content of this files